source/replication/server-side-replication-resynchronize-remote.rst

```diff
@@ -0,0 +1,321 @@
+<alias>` for each MinIO cluster. You should replace these values with the
+appropriate aliases for the MinIO deployments on which you are configuring
+bucket replication. These examples assume that the specified aliases have the
+necessary permissions for creating policies and users on both deployments. See
+:ref:`minio-users` and :ref:`MinIO Policy Based Access Control <minio-policy>`
+for more complete documentation on MinIO users and policies respectively.
+
+The following code creates the user and policies necessary for
+resynchronization of data from ``Alpha`` to ``Baker``. Replace the password
+``LongRandomSecretKey`` with a long, random, and secure secret key as per
+your organizations best practices for password generation.
+
+.. code-block:: shell
+   :class: copyable
+
+   wget -O - https://docs.min.io/minio/baremetal/examples/ReplicationAdminPolicy.json | \
+   mc admin policy add Alpha ReplicationAdminPolicy /dev/stdin
+   mc admin user add Alpha alphaReplicationAdmin LongRandomSecretKey
+   mc admin policy set Alpha ReplicationAdminPolicy user=alphaReplicationAdmin
+
+   wget -O - https://docs.min.io/minio/baremetal/examples/ReplicationRemoteUserPolicy.json | \
+   mc admin policy add Baker ReplicationRemoteUserPolicy /dev/stdin
+   mc admin user add Baker bakerReplicationRemoteUser LongRandomSecretKey
+   mc admin policy set Baker ReplicationRemoteUserPolicy user=bakerReplicationRemoteUser
+
+You can skip this step if
+both deployments already have users with the necessary :ref:`permissions
+<minio-bucket-replication-serverside-resynchronize-permissions>`.
+
+2) Configure ``mc`` Replication Admin Access
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the :mc-cmd:`mc alias set` command to add a replication-specific alias for
+the ``Alpha`` deployment:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc alias set AlphaReplication HOSTNAME AlphaReplicationAdmin LongRandomSecretKey
+
+3) Start the Resynchronization Procedure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the :mc-cmd:`mc replicate resync` command to begin the resynchronization
+process:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc replicate resync --remote-bucket "arn:minio:replication::UUID:BUCKET" alpha/BUCKET
+
+- Replace the ``--remote-bucket`` value with the ARN of the ``BUCKET`` on the
+  ``Baker`` deployment. 
+
+  Use :mc-cmd:`mc admin bucket remote ls` to retrieve the replication target
+  ARN.
+
+- Replace the ``BUCKET`` with the name of the bucket on the source MinIO
+  deployment.
+
+The command returns a resynchronization job ID indicating that the process has
+begun.
+
+4) Monitor Resynchronization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use any of the following methods for tracking resynchronization progress:
+
+- Run :mc-cmd:`mc replicate status` on both ``Alpha`` and ``Baker`` to track
+  the sent and received replication data.
+
+- Run :mc-cmd:`mc ls -r --versions ALIAS | wc -l <mc ls>` on
```

Comment:
may want to just recommend `ls` over versions in a bucket instead of the entire namespace

Reply:
This is not going to work for our users at the scale they might deploy things, we need to provide a different mechanism here.  We should leave it as is for now and users shouldn't be relying on `mc ls` to validate if they have content or not.